hack/jsonschema/main.go

```diff
@@ -5,6 +5,8 @@ import (
 	"os"
 )
 
+type obj = map[string]interface{}
```

Comment:
Not sure why I need to do this, but I can't run main.go otherwise - `obj` registers as undeclared name